Tasks/Rabushenka/Tasks7/HomWr-7/episode_data.py

```diff
@@ -0,0 +1,59 @@
+Dictionary_football = [["form kit", 120, 10, 1], ["balls", 50, 7, 2],["football paraphernalia", 30, 12, 3]]
+Dictionary_hockey = [["skates", 80, 10, 1], ["hockey stick", 20, 10, 2],["protective equipment", 340, 5, 3]]
+Dictionary_tennis = [["racket", 85, 20, 1], ["tennis balls", 10, 100, 2],["form kit", 75, 5, 3]]
+Dictionary_other = [["skiing", "price:90$", 6, 1], ["parachutes", 230, 3, 2]]
+
+basket = []
+
+def get_all_1():
+    return Dictionary_football
+
+def get_all_2():
+    return Dictionary_hockey
+
+def get_all_3():
+    return Dictionary_tennis
+
+def get_all_4():
+    return Dictionary_other
```

Comment:
you hard-coded the categories, which means that you have to write a specific code to work with a specific category, it's no good, would be better to use some common methods to access any category via some generic logic